workflows/sock-shop-demo/usingCMDProbe/workflow.yaml

```diff
@@ -32,6 +34,8 @@ spec:
             template: pod-cpu-hog
         - - name: pod-delete
             template: pod-delete
+          - name: pod-network-loss
```

Comment:
Can we add some suffix/prefix to explicitly mentioned that "this to run the experiment" OR creation of the chaos engine?